False errors "mediawiki/valid-package-file-require" when using Windows

I develop on Windows and after a recent update I started getting "mediawiki/valid-package-file-require" errors all over the place. Apparently it wants me to use backslashes, which probably won't actually work.

For example, a snippet from the `grunt` output in DiscussionTools:
```
C:\mediawiki\extensions\DiscussionTools\tests\qunit\modifier.test.js
    9:14  error  Incorrect file path in require(): use ..\cases\modified.json instead  mediawiki/valid-package-file-require
   71:14  error  Incorrect file path in require(): use ..\cases\reply.json instead     mediawiki/valid-package-file-require
  118:14  error  Incorrect file path in require(): use ..\cases\unwrap.json instead    mediawiki/valid-package-file-require

C:\mediawiki\extensions\DiscussionTools\tests\qunit\parser.test.js
    9:14  error  Incorrect file path in require(): use ..\cases\timestamp-regex.json instead       mediawiki/valid-package-file-require
   23:14  error  Incorrect file path in require(): use ..\cases\timestamp-parser.json instead      mediawiki/valid-package-file-require
   39:14  error  Incorrect file path in require(): use ..\cases\timestamp-parser-dst.json instead  mediawiki/valid-package-file-require
   61:11  error  Incorrect file path in require(): use ..\cases\comments.json instead              mediawiki/valid-package-file-require
  103:11  error  Incorrect file path in require(): use ..\cases\transcluded.json instead           mediawiki/valid-package-file-require
  142:14  error  Incorrect file path in require(): use ..\cases\authors.json instead               mediawiki/valid-package-file-require
```

--- a/src/rules/valid-package-file-require.js
+++ b/src/rules/valid-package-file-require.js
@@ -15,7 +15,9 @@ function getFullRelativeFilePath( name, context ) {
 		dotSlashPrefixIfMissing( name ),
 		{ paths: [ contextDirPath ] }
 	);
-	const relativePath = path.relative( contextDirPath, absolutePath );
+	const relativePath = path.relative( contextDirPath, absolutePath )
+		// On Windows systems, this returns a path with backslashes (\), change them to slashes (/)
+		.replace( /\\/g, '/' );
 
 	return dotSlashPrefixIfMissing( relativePath );
 }

